src/ccl_background.c

```diff
@@ -714,23 +714,39 @@ void ccl_cosmology_distances_from_input(ccl_cosmology * cosmo, int na, double a[
 INPUT: cosmology, scale factor array, growth array, growth rate array
 TASK: if not already there, create growth splines with the input arrays and store them.
 */
-void ccl_cosmology_growth_from_input(ccl_cosmology* cosmo, int na, double a[], double growth_arr[], double fgrowth_arr[], int* status)
+void ccl_cosmology_growth_from_input(ccl_cosmology* cosmo, int na, double a[],
+                                     double growth_arr[], double fgrowth_arr[], int* status)
 {
   int chistatus;
   if (cosmo->computed_growth)
     return;
 
+  double *growth_normed = NULL;
   gsl_spline * growth = gsl_spline_alloc(cosmo->spline_params.A_SPLINE_TYPE, na);
   gsl_spline * fgrowth = gsl_spline_alloc(cosmo->spline_params.A_SPLINE_TYPE, na);
+  double growth0 = growth_arr[na-1];
```

Comment:
We need a check here or in the python code that ensure this element is at or close to `a=1`.

Reply:
I think this check is done in python. Can you add a comment here to that effect assuming I understood the code correctly?

Reply:
Yep, that's right. Added.